new-dti-website/src/app/apply/page.tsx

```diff
@@ -1,21 +1,30 @@
 'use client';
 
+import { useState } from 'react';
 import ApplicationTimeline from '../../../components/apply/ApplicationTimeline';
 import config from '../../../config.json';
+import RoleDescriptions from '../../../components/apply/RoleDescription';
+import RedBlob from '../../../components/blob';
 
 const ApplyHero = () => (
```

Comment:
I think it'd be a great idea to, instead of implementing these within page.tsx, create new components for these "sections". This would make it a lot more readable for other users, especially if we might have to make specific changes. It's pretty hard having to find a single component or section on a single page. (I still have to work on this to as well haha). The more reusable the better!